File: src/main/kotlin/org/team5499/frc2019/Main.java
package org.team5499.frc2019;

import edu.wpi.first.wpilibj.RobotBase;
import edu.wpi.first.wpilibj.TimedRobot;

import org.team5499.frc2019.Robot;

public final class Main {

    private Main() {}

    public static void main() {
        RobotBase.startRobot(Robot::new);
    }

}

File: src/main/kotlin/org/team5499/frc2019/Main.kt
package org.team5499.frc2019

import edu.wpi.first.wpilibj.RobotBase

import java.util.function.Supplier

public class Main {

    /**
     * Main initialization function. Do not perform any initialization here.
     *
     * <p>If you change your main robot class, change the parameter type.
     */
    companion object {
        @JvmStatic
        public fun main(args: Array<String>) {
            RobotBase.startRobot(Supplier<Robot> { Robot() })
        }
    }
}
